### PR TITLE
Chainflip tvl

### PR DESCRIPTION
**Pull request by CumpsD**
This is an update to an existing adapter to include the deployed liquidity in the system as TVL.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/chainflip exports TVL: 
        
 
 
    ethereum-staking          54.65 M
    staking                   54.65 M
    chainflip                 4.70 M
    ethereum                  0
    
    total                    4.70 M

**Comment by llamatester** [comment]
The adapter at projects/chainflip exports TVL: 
        
 
 
    ethereum-staking          54.65 M
    staking                   54.65 M
    chainflip                 4.53 M
    ethereum                  0
    
    total                    4.53 M

**Comment by llamatester** [comment]
The adapter at projects/chainflip exports TVL: 
        
 
 
    ethereum-staking          54.65 M
    staking                   54.65 M
    chainflip                 5.09 M
    ethereum                  0
    
    total                    5.09 M

File: projects/chainflip/index.js [hunk 1]
```diff
@@ -1,13 +1,55 @@
...
+            unavailableAmount
+        }
     }
+}`
+
+const endpoint = 'https://cache-service.chainflip.io/graphql'
+
+async function tvl(api) {
+  // Call GraphQL and get tokens, add each to balance
+  const { allPools: { nodes }, allBoostPools: { nodes: bNodes } } = await graphQuery(endpoint, poolsDataQuery);
```
[the remark this answers is not in the document]
Comment (CumpsD) [reply projects/chainflip/index.js:32]: They are 2 different types of liquidity, normal pools is money used to facilitate swaps, while a boost pool is liquidity used to front a BTC swap for a user

**Comment by CumpsD** [comment]
@g1nt0ki thanks for the refactor, didnt know that helper :)

**Pull request merged**